bip-update-process.md

```diff
@@ -0,0 +1,595 @@
+  implemented, and applications using Bitcoin can be compliant with it. Specification BIPs must contain a specification
+  section as well as contain or link to a reference implementation.
+* An **Informational BIP** describes a Bitcoin design issue, provides general guidelines, or other information to the
+  Bitcoin community, but does not propose a new feature.
+* A **Process BIP** describes a process surrounding Bitcoin, or proposes a change to (or an event in) a process. Process
+  BIPs are like Specification BIPs, but apply to areas other than the Bitcoin protocol itself. They may propose an
+  implementation, but not to Bitcoin's codebase; they often require community consensus; unlike Informational BIPs, they
+  are more than recommendations, and users are typically not free to ignore them. Examples include procedures,
+  guidelines, and changes to the decision-making process. Any meta-BIP is also considered a Process BIP.
+
+## Workflow
+
+The BIP process starts with a new idea for Bitcoin. Each potential BIP must have an author (or multiple
+authors)—someone who writes the BIP, gathers feedback, shepherds the discussion in the appropriate forums, and
+finally recommends a mature proposal to the community.
+
+![Status Diagram](bip-update-process/status-diagram.png "Status Diagram for the BIP Workflow")
+
+### Ideation
+
+After having an idea, the author should evaluate whether it is meets the criteria to become a BIP, as described in
+this BIP. The idea must be of interest to the broader community or relevant to multiple software projects. Small
+improvements and matters concerning only a single project usually do not require standardization and should instead be
+brought up to the relevant project directly.
+
+The author should first research whether an idea has been considered before. Ideas in Bitcoin are often rediscovered,
+and prior related discussions may inform the author what issues may arise in its progression. After some investigation,
+the novelty of an idea can be tested by posting about it to the [Bitcoin development mailing
+list](https://groups.google.com/g/bitcoindev).
+
+Vetting an idea publicly before investing the time to formally describe the idea is meant to save both the author and
+the broader community time. Not only may someone point out relevant discussion topics that were missed in the author’s
+research, or that an idea is guaranteed to be rejected based on prior discussions, but describing an idea publicly also
+tests whether it is of interest to more people beside the author. After establishing that the idea may be of interest
+to the Bitcoin community, the author should work on drafting a BIP.
+
+It is highly recommended that a single BIP contain a single key proposal or new idea. The more focused the BIP, the more
+successful it tends to be. If in doubt, split your BIP into several well-focused ones.
```

Comment:
```suggestion
successful it tends to be. If in doubt, the BIP should be separated into several well-focused ones.
```
(The rest of the text does not address the reader directly.)

Maybe add: "in particular if it may help the reuse of subcomponents of the idea."